doc/dnf-1_vs_dnf-2.rst

```diff
@@ -0,0 +1,70 @@
+..
+  Copyright (C) 2014-2016 Red Hat, Inc.
```

Comment:
not really. you can, but you should consider to not license documentation under GPLv2+ (there is GFDL, but it's not allowed in Debian, there are more like CC-BY-SA and others). But basically GPLv2+ for documentation is painful.